tests/test_acasclient.py

```diff
@@ -1797,4 +1595,299 @@ def test_044_author_and_role_apis(self):
         with self.assertRaises(requests.HTTPError) as context:
             acas_user_author_role['userName'] = test_username
             user_client.update_author_roles(author_roles_to_delete=[acas_user_author_role])
-        self.assertIn('500 Server Error', str(context.exception))
\ No newline at end of file
+        self.assertIn('500 Server Error', str(context.exception))
+
+
+
+def experiment_load_test(data_file_to_upload, dry_run_mode, self):
+    response = self.client.\
+        experiment_loader(data_file_to_upload, "bob", dry_run_mode)
+    self.assertIn('results', response)
+    self.assertIn('htmlSummary', response['results'])
+    self.assertIn('errorMessages', response)
+    self.assertIn('hasError', response)
+    self.assertIn('hasWarning', response)
+    self.assertIn('transactionId', response)
+    if dry_run_mode:
+        self.assertIsNone(response['transactionId'])
+    else:
+        self.assertIsNotNone(response['transactionId'])
+    return response
+
+def csv_to_txt(data_file_to_upload, self):
+    # Get the file name but change it to .txt
+    file_name = data_file_to_upload.name
+    file_name = file_name.replace(".csv", ".txt")
+    temp_file_path = Path(self.tempdir, file_name)
+    # Change the delim to the new delim
+    with open(data_file_to_upload, 'r') as f:
+        with open(temp_file_path, 'w') as f2:
+            for line in f:
+                f2.write(line.replace(',', "\t"))
+    return temp_file_path
+
+class TestExperimentLoader(BaseAcasClientTest):
```

Comment:
I like splitting these out into a separate test, but don't they require some compound loading to happen first?
It would be better to have some `setUp` logic that confirms the necessary compound lots exist, and registers them if not, that way this `TestExperimentLoader` test case can be run standalone.

Reply:
Did major refactor to fix this.  Added setUpClass and tearDownClass as well as a few decorators for ensuring the database was in the proper state for the test.